data/campaigns/Eastern_Invasion/scenarios/17b_All-In.cfg

```diff
@@ -0,0 +1,1663 @@
+                [/not]
+            [/filter_recall]
+        [/modify_unit]
+        [set_extra_recruit]
+            id=Halric,Halrad,Halrod
+            extra_recruit={REGULAR_RECRUIT_LIST}
+        [/set_extra_recruit]
+
+        {RECALL_XY Konrad 28 22}
+    [/event]
+
+    [event]
+        name=start
+        {RECALL_XY Gweddry 29 23}
+        {RECALL_XY Dacyn 27 23}
+        {RECALL_XY Owaec 29 22}
+        {RECALL_XY Addogin 28 21}
+        {RECALL_XY (Hahid al-Ali) 28 21}
+        {RECALL_XY Terraent 28 21}
+        {RECALL_XY Grug 27 22}
+        {RECALL_XY Dolburras 27 22}
+        {RECALL_XY Yannic 28 23}
+        {RECALL_XY Gaennell 28 22} # any open spot near the king
+
+        [message]
+            speaker=Halrad
+            message= _ "My forces are in position at the northern wall; awaiting your orders."
+        [/message]
+        [message]
+            speaker=Halric
+            message= _ "Eastern flank reclaimed. I stand ready to launch the assault."
+        [/message]
+        [message]
+            speaker=Halrod
+            message= _ "The western barricades are manned and ready - it’s payback time."
+        [/message]
+        [if]
+            [have_unit]
+                id=Owaec
+            [/have_unit]
+
+            [then]
+                [message]
+                    speaker=Owaec
+                    message= _ "Now is the time to strike! In the name of the Clans, Mal-Ravanal will be ended this day!"
+                [/message]
+            [/then]
+
+            [else]
+                [message]
+                    speaker=Konrad
+                    message= _ "Now is the time to strike! By the might of all Wesnoth, Mal-Ravanal will be ended this day!"
+                [/message]
+            [/else]
+        [/if]
+
+        {SCROLL_TO 61 1}
+        [delay]
+            time=200
+        [/delay]
+
+        {SCROLL_TO 47 34}
+        [delay]
+            time=200
+        [/delay]
+
+        {SCROLL_TO 1 27}
+        [delay]
+            time=200
+        [/delay]
+
+        {SCROLL_TO 1 1}
+        [delay]
+            time=200
+        [/delay]
+
+		[message]
+			speaker=Dacyn
+			message= _ "These liches are channeling some dark magic to hide themselves. (cough)(cough) Even if Mal-Ravanal is here, I will not be able to (cough) reveal them until we have killed enough of these liches to break their spell. (cough)"
+		[/message]
+
+        [if]
+            [have_unit]
+                id=Gweddry
+            [/have_unit]
+
+            [then]
+                [message]
+                    speaker=Gweddry
+                    message= _ "Then that’s what we’ll have to do. And we will have to do it quickly, before Mal-Ravanal’s reinforcing armies overwhelm us."
+                [/message]
+                [message]
+                    speaker=Konrad
+                    message= _ "Stride fearless upon the plain of battle, for today we reclaim our legacy! For Wesnoth!!"
```

Comment:
'Stride fearlessly'

Also, double exclamation mark.

Reply:
I like the more archaic sound of "fearless", but not a big deal either way.

Reply:
Hmm, 'fearless' sounds more modern to me, since I notice this pattern of speech (using adjectives where adverbs should be used) more common in `en_US`, particularly in informal contexts. And given it's the King speaking, I'd favour formality.